theoplayer/getting-started/01-sdks/03-ios/00-getting-started.mdx

```diff
@@ -0,0 +1,196 @@
+# Getting started on iOS/tvOS
+
+## Introduction
+
+This guide covers the steps to obtain and setup the core THEOplayer SDK targeting a native iOS/tvOS application. It also provides information and usage of the commonly used APIs.
+
+## Setup
+
+### THEOplayer license
+
+THEOplayer requires a valid THEOplayer license to be configured in order to access the SDK APIs (with the exception of some whitelisted sources hosted by our CDN). Request your license via [https://portal.theoplayer.com](https://portal.theoplayer.com).
+
+After creating your license, you can copy its license string to your clipboard as demonstrated in the screenshot below.
+
+![](https://cdn.theoplayer.com/images/git/theoplayer-ios-sdk-license-string.png)
+
+> **Note:** It's possible to add multiple platforms in a single license ie. iOS & tvOS.
+
+Later in the guide we explain how to configure the license ([in this section](#adding-the-theoplayer-license)).
+
+### Installation
+
+The THEOplayer SDK is published on the following package managers:
+
+#### <span style={{ fontSize: '1.25rem' }}>Cocoapods</span>
+
+> **_It's considered good practice to run `pod repo update` before adding new pods to your Podfile.
+This will help find newly added pods by updating the list of references to the public spec repos.
+If you just installed Cocoapods or are already up-to-date, then this step can be skipped._**
+
+<br/>
+Add the following entry to your Podfile:
+
+```ruby
+target 'MyApp' do
+  pod 'THEOplayerSDK-core', '~> 8' # at the time of writing, v8 is the latest major version.
+end
+```
+
+> You can skip specifying the version to get the latest published version of THEOplayerSDK.
+
+Then, using the terminal navigate to the directory where your Podfile is located and run:
+
+```bash
+pod install
+```
+
+If it's the first time that your project integrates with Cocoapods, then after this step you should stop using the `xcodeproj` file and instead start using the `xcworkspace` file generated by Cococapods.
+
+#### <span style={{ fontSize: '1.25rem' }}>Swift Package Manager</span>
+
+Navigate to your Xcode project, and select the `Package Dependencies` tab, then click on the `+` button to add a new dependency. Then in the search bar, add the following URL https://github.com/THEOplayer/theoplayer-sdk-apple. Choose the version and click on `Add Package`.
+
+The following image illustrates what it should look like:
+
+![](../../../../theoplayer/assets/img/ios-getting-started-1.png)
+
+> **_If Xcode fails to fetch the THEOplayer package, navigate in Terminal to the root directory of your project and run `xcodebuild -resolvePackageDependencies`. This will re-attempt to fetch the packages specified in the project. Restart your Xcode and the package dependencies should be in place._**
```

Comment:
```suggestion
:::tip

If Xcode fails to fetch the THEOplayer package, navigate in Terminal to the root directory of your project and run `xcodebuild -resolvePackageDependencies`. This will re-attempt to fetch the packages specified in the project. Restart your Xcode and the package dependencies should be in place.

:::
```